.expeditor/config.yml

```diff
@@ -44,15 +49,29 @@ pipelines:
       description: Deploy the website
       definition: .expeditor/website.pipeline.yml
       trigger: pull_request
-  - habitat/build
   - release_habitat:
       description: Habitat release process
   - end_to_end:
       description: Habitat long running end to end tests - always runs on DEV environment
 
+staging_areas:
+  - post_merge:
+      workload: pull_request_merged:habitat-sh/habitat:master:*
+
 subscriptions:
   # These actions are taken, in order they are specified
+  - workload: pull_request_merged:{{github_repo}}:{{release_branch}}:*
+    actions:
+      - trigger_pipeline:release_habitat:
+
   - workload: buildkite_build_passed:{{agent_id}}:release_habitat:*
```

Comment:
if the release pipeline is successful, the E2E pipeline will automatically run

Reply:
When do you unlock your staging area in the success case?

Reply:
Fixed, thanks for the explanation!